src/poliastro/plotting/orbit/backends/base_backend.py

```diff
@@ -0,0 +1,232 @@
+"""A module implementing the base class for a orbit plotter backend."""
+
+
+class _OrbitPlotterBackend:
+    """A base class for implementing new orbit plotter backends."""
+
+    def __init__(self, scene, name, ref_units):
+        """Initialize the orbit plotter backend.
+
+        Parameters
+        ----------
+        scene : object
```

Comment:
What types are allowed for `scene`? (This would help understanding the implementation I believe)